packages/codegen-doc/src/print-operation.ts

```diff
@@ -174,3 +177,15 @@ export function printOperations(
     return undefined;
   }
 }
+
+/**
+ * Prefix mutations with the type of the mutation.
+ * @param operationName The original operation name.
+ */
+export function printPrefixedMutationName(operationName: string): string {
+  const mutationType = Doc.MUTATION_TYPES.find(type => operationName.endsWith(upperFirst(type)));
+  if (mutationType) {
+    return lowerFirst(`${mutationType}${upperFirst(operationName.replace(upperFirst(mutationType), ""))}`);
```

Comment:
For `projectUpdateUpdate` this would actually remove the `Update` in the middle and not the last part. It is probably safe, but wondering if it would be safer to discard the last `mutationType.length` characters instead?

Reply:
That's an excellent point. It would indeed be safer to ensure the verb is indeed at the end of the operation name. Will update.